components/meter/README.md

```diff
@@ -0,0 +1,49 @@
+# Meter Components
+
+* Note this is considered as progress indicator in design.
+
+## d2l-meter-linear
+<img src="../../screenshots/meter/d2l-meter-linear-no-progress.png">
+<img src="../../screenshots/meter/d2l-meter-linear-has-progress.png">
+<img src="../../screenshots/meter/d2l-meter-linear-completed.png">
+
+### Usage
+
+Import
+```js
+import '/@brightspace-ui/core/components/meter/meter-linear.js';
+```
+
+Then add the `d2l-meter-linear`, provide values for the properties, `value` and `max`.
+
+```html
+<d2l-meter-linear value="30" max="100"></d2l-meter-linear>
+```
+
+***Properties:***
+
+* `value` (required): The current number of units that have been marked as completed. Should be a positive, non-zero number that is less than or equal to `max`
+* `max` (required): Max number of units that is being measured by this meter. For results this should be a positive, non-zero number
```

Comment:
This got me thinking... should we have defaults for these so that they aren't required? Not sure.

Reply:
I think having defaults for these not useful. Any real use should use these.

Reply:
Fair enough. I think the native `<progress>` defaults the `max` to 1 and `value` is undefined which puts it into a weird indeterminate state that we don't support here.